src/App.js

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
-import './App.css';
 
-const App = () => (
-  <h1>Count: 0</h1>
-);
+export class App extends React.Component {
+  state = {
+    count: 0,
+  }
 
-export default App;
+  addCount = () => {
```

Comment:
текущее название подразумевает что метод добавляет каунтер (на страницу), а не увеличивает его значение
```suggestion
  incrementCounter = () => {
```